src/beacon-api/app/utils/field_mappings.py

```
@@ -0,0 +1,106 @@
+"""
+Field1–20 → human-readable label mappers per device category.
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)